development.ini

```diff
@@ -15,6 +15,7 @@ jujugui.combine = true
 jujugui.gzip = true
 jujugui.cachebuster = dev
 jujugui.socketTemplate = /environment/$uuid/api
+jujugui.embedded = true
```

Comment:
Let's not call this embedded, b/c it's not necessarily true for all embedded contexts.